assert_writeln_magic.d

```diff
@@ -236,72 +249,70 @@ Tries to lower EqualExpression in AssertExpressions of Unittest blocks to commen
     }
 
     foreach (file; files)
+    {
         if (!ignoredFiles.any!(x => file.name.canFind(x)))
-            file.name.parseFile;
+        {
+            // single files
+            if (inputDir.length == 0)
+                parseFile(file.name, outputDir);
+            else
+                parseFile(file.name, file.name.rebasePath(inputDir, outputDir));
```

Comment:
btw I just kept this support because it might be nice "in a possible future". It's not needed anymore as the Makefile does the invocations with a single file.